lib/datadog/tracing/contrib/propagation/sql_comment.rb

```diff
@@ -17,11 +17,11 @@ def self.annotate!(span_op, mode)
 
           # Inject span_op and trace_op instead of TraceDigest to improve memory usage
           # for `disabled` and `service` mode
-          def self.prepend_comment(sql, span_op, trace_op, mode)
+          def self.prepend_comment(sql, span_op, trace_op, mode, peer_service)
```

Comment:
The span has already be injected into the method. No need to  add another parameter

Reply:
Good catch, done